### ISSUE TITLE
Wrong torch version?

### ISSUE THREAD

**Issue by ZJaume** [jpuigcerver/PyLaia#15]
I was trying to resume a training and I switched to the `'refactor_kws_egs_master'` and `pylaia-htr-train-ctc` prompted an `ImportError`:
```
Traceback (most recent call last):
  File "/labos/alumnos/jaizaber/workw/RDNN-HTR-PY/bin/pylaia-htr-train-ctc", line 4, in <module>
    __import__('pkg_resources').run_script('laia==0.1.0', 'pylaia-htr-train-ctc')
  File "/labos/alumnos/jaizaber/workw/RDNN-HTR-PY/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 658, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/labos/alumnos/jaizaber/workw/RDNN-HTR-PY/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 1438, in run_script
    exec(code, namespace, namespace)
  File "/labos/alumnos/jaizaber/workw/RDNN-HTR-PY/lib/python3.6/site-packages/laia-0.1.0-py3.6.egg/EGG-INFO/scripts/pylaia-htr-train-ctc", line 13, in <module>
    import laia.common.logging as log
  File "/labos/alumnos/jaizaber/workw/RDNN-HTR-PY/lib64/python3.6/site-packages/laia-0.1.0-py3.6.egg/laia/__init__.py", line 21, in <module>
    import laia.data
  File "/labos/alumnos/jaizaber/workw/RDNN-HTR-PY/lib64/python3.6/site-packages/laia-0.1.0-py3.6.egg/laia/data/__init__.py", line 4, in <module>
    from laia.data.image_data_loader import ImageDataLoader
  File "/labos/alumnos/jaizaber/workw/RDNN-HTR-PY/lib64/python3.6/site-packages/laia-0.1.0-py3.6.egg/laia/data/image_data_loader.py", line 5, in <module>
    from laia.data.padding_collater import PaddingCollater
  File "/labos/alumnos/jaizaber/workw/RDNN-HTR-PY/lib64/python3.6/site-packages/laia-0.1.0-py3.6.egg/laia/data/padding_collater.py", line 11, in <module>
    from torch.utils.data.dataloader import numpy_type_map
ImportError: cannot import name 'numpy_type_map'
```
That's because `numpy_type_map` was removed on latest version of Torch. I downgraded Torch to 1.0.1 and Torchvision to 0.2.2 and it worked. I hope this could help to someone who could have the same problem.

**Comment by jpuigcerver** [comment]
Thanks for letting me know. I need to update PyLaia to PyTorch 1.1

**Comment by jpuigcerver** [comment]
Fixed with ef2c59d4dafa386309556a2ab8dcf53707103eff.
